third_party/nvfuser/csrc/ir_internal_nodes.h

```diff
@@ -1394,6 +1395,7 @@ class TORCH_CUDA_CU_API IterDomainBuilder {
   bool is_padded_dimension_ = false;
   c10::optional<int64_t> padded_to_size_ = c10::nullopt;
   bool is_mma_swizzled_ = false;
+  bool is_merged_from_broadcast_ = false;
```

Comment:
It looks like this is only used in lowering. It should probably be added to the concretized broadcast information, not as a member to IterDomain.

Reply:
> what's the perf delta for the softmax case?
> 
For the case in https://github.com/csarofeen/pytorch/issues/2125
Before this fix was 340~370 GB/s on A100-80G, after this fix increased to 1.19e+03 GB/s on the same GPU.

Reply:
Very nice.

Reply:
> It looks like this is only used in lowering. It should probably be added to the concretized broadcast information, not as a member to IterDomain.

This parameter is deleted in the revised version. The infomation can be reconstructed by itering over exprs between leaf domains and root domain.
```
all_exp = DependencyCheck::getAllExprsBetween(
        {from.begin(), from.end()},
        {out_tv->domain()->domain().begin(), out_tv->domain()->domain().end()});
```